helloworld-mutual-ssl-secured/configure-certs.cli

```diff
@@ -0,0 +1,19 @@
+#configure a key-store in the Elytron subsystem. The path to the keystore file doesn’t actually have to exist yet.
+/subsystem=elytron/key-store=clientKS:add(path=client.keystore.P12, relative-to=jboss.server.config.dir, credential-reference={clear-text=secret}, type=PKCS12)
+
+#generate a new key pair which will be used later to extract the certificate. This is an RSA key of size 2048. CN must be quickstartUser for the key
+/subsystem=elytron/key-store=clientKS:generate-key-pair(alias=quickstartUser, algorithm=RSA, key-size=2048, validity=365, credential-reference={clear-text=secret}, distinguished-name="cn=quickstartUser")
+
+#Export the certificate to a file
```

Comment:
Export the client's certificate to a file, clientCert.crt.

Reply:
fixed!